vpdq/setup.py

```diff
@@ -1,40 +1,32 @@
 from setuptools import setup
 from setuptools.extension import Extension
-from setuptools.command.build_py import build_py
+from setuptools.command.build_ext import build_ext                                     
 import sys
 import subprocess
 import os
 import numpy
 from pathlib import Path
 
-read_me = Path.cwd() / Path("README.md")
+DIR = Path(__file__).parent
+read_me = DIR / Path("vpdq/README.md")
 long_description = read_me.read_text()
+version = (DIR / "vpdq/version.txt").read_text(encoding="utf-8").strip()
 
-
-class Build(build_py):
-    """Customized setuptools build command - builds protos on build."""
-
+class build_ext(build_ext):
     def run(self):
-        os.chdir("cpp")
-        command = ["mkdir", "build"]
-        subprocess.call(command)
-        os.chdir("build")
-        command = ["cmake", ".."]
-        if subprocess.call(command) != 0:
-            sys.exit(-1)
+        os.chdir("vpdq/cpp")
         command = ["make"]
         if subprocess.call(command) != 0:
```

Comment:
Consider using `check_call` instead.

I generally recommend against sys.exit vs throwing exceptions, because at least you can debug from the stack trace